### DISCUSSION TITLE
Amethyst 0.20.0 Beta 2

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1501] [Announcements]
# Notable Changes
* Fix a frequent crash on some types of long running application launches.

## What's Changed
* Remove observer for current notification on errors by @ianyh in https://github.com/ianyh/Amethyst/pull/1500

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.20.0b...v0.20.0b2

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.20.0b2'>Amethyst 0.20.0 Beta 2</a>.</em>

**Comment by yangwao** [comment]
Hey, seems this build is broke, I would unpublish best @ianyh ? 

Updated to beta3, works! 

https://github.com/ianyh/Amethyst/releases/tag/v0.20.0b3

**Reply by ianyh** [reply]
Yeah, I made a dumb mistake in beta 2. Deleted the artifacts for it.
